avaframe/out3Plot/outAIMEC.py

```diff
@@ -117,13 +117,92 @@ def visuRunout(rasterTransfo, resAnalysis, plim, newRasters, cfgPath, cfgFlags):
     dataPressure = newRasters['newRasterPressure']
     rasterdataPres = dataPressure[0]
     runout = resAnalysis['runout'][0]
-    pCrossAll = resAnalysis['pCrossAll']
+    pCrossMaxAll = resAnalysis['pCrossMaxAll']
+    pCrossMeanAll = resAnalysis['pCrossMeanAll']
+    fdCrossMaxAll = resAnalysis['fdCrossMaxAll']
+    fdCrossMeanAll = resAnalysis['fdCrossMeanAll']
+    fvCrossMaxAll = resAnalysis['fvCrossMaxAll']
+    fvCrossMeanAll = resAnalysis['fvCrossMeanAll']
+
+    ############################################
+    # Figure: Analysis runout
+    fig = plt.figure(figsize=(pU.figW*2, pU.figH))
+    ax1 = plt.subplot(131)
+
+    ax1.plot(pCrossMaxAll[0, :], s, '--k', label='Max Reference')
+    ax1.plot(pCrossMeanAll[0, :], s, '-k', label='Mean Reference')
+    ax1.plot(pCrossMaxAll[1, :], s, '--b', label='Max Simulation')
+    ax1.plot(pCrossMeanAll[1, :], s, '-b', label='Mean Simulation')
+
+    ax1.set_title('Pressure distribution along the path')
+    ax1.legend(loc=4)
+    ax1.set_ylabel('l [m]')
+    ax1.set_ylim([s.min(), s.max()])
+    ax1.set_xlim(auto=True)
+    ax1.set_xlabel('$P(s)$ [kPa]')
+
+    ax2 = plt.subplot(132)
+
+    ax2.plot(fdCrossMaxAll[0, :], s, '--k', label='Max Reference')
+    ax2.plot(fdCrossMeanAll[0, :], s, '-k', label='Mean Reference')
+    ax2.plot(fdCrossMaxAll[1, :], s, '--b', label='Max Simulation')
+    ax2.plot(fdCrossMeanAll[1, :], s, '-b', label='Mean Simulation')
+
+    ax2.set_title('Flow Depth distribution along the path')
+    ax2.legend(loc=4)
+    ax2.set_ylabel('l [m]')
+    ax2.set_ylim([s.min(), s.max()])
+    ax2.set_xlim(auto=True)
+    ax2.set_xlabel('$h(s)$ [m]')
+
+    ax3 = plt.subplot(133)
+
+    ax3.plot(fvCrossMaxAll[0, :], s, '--k', label='Max Reference')
+    ax3.plot(fvCrossMeanAll[0, :], s, '-k', label='Mean Reference')
+    ax3.plot(fvCrossMaxAll[1, :], s, '--b', label='Max Simulation')
+    ax3.plot(fvCrossMeanAll[1, :], s, '-b', label='Mean Simulation')
+
+    ax3.set_title('Flow velocity distribution along the path')
+    ax3.legend(loc=4)
+    ax3.set_ylabel('l [m]')
+    ax3.set_ylim([s.min(), s.max()])
+    ax3.set_xlim(auto=True)
+    ax3.set_xlabel('$v(s) [m.s^{-1}]$')
+
```

Comment:
I suggest making a small internal function that does the plotting and styling which you can call 3 time with the input
`XXCrossMaxAll, label, title`